libs/payments/paypal/src/lib/client.spec.ts

```diff
@@ -1,41 +1,38 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-import nock from 'nock';
 import { faker } from '@faker-js/faker';
+import nock from 'nock';
 
 import { PayPalClient } from './client';
-
-import { PayPalClientError } from './error';
-
-import {
-  BAUpdateOptions,
-  DoReferenceTransactionOptions,
-  PaypalMethods,
-  RefundTransactionOptions,
-  RefundType,
-  TransactionSearchOptions,
-} from './types';
-
 import {
+  PAYPAL_IPN_ROUTE,
+  PAYPAL_NVP_ROUTE,
   PAYPAL_SANDBOX_BASE,
   PAYPAL_SANDBOX_IPN_BASE,
-  PAYPAL_NVP_ROUTE,
-  PAYPAL_IPN_ROUTE,
-  PLACEHOLDER_URL,
   PAYPAL_VERSION,
+  PLACEHOLDER_URL,
 } from './constants';
-
+import { PayPalClientError } from '@fxa/shared/error';
 import {
   NVPBAUpdateTransactionResponseFactory,
   NVPDoReferenceTransactionResponseFactory,
+  NVPErrorFactory,
   NVPErrorResponseFactory,
   NVPRefundTransactionResponseFactory,
   NVPResponseFactory,
   NVPSetExpressCheckoutResponseFactory,
   NVPTransactionSearchResponseFactory,
 } from './factories';
+import {
+  BAUpdateOptions,
+  DoReferenceTransactionOptions,
+  NVPErrorSeverity,
+  PaypalMethods,
```

Comment:
Probably gone too far on naming, but we have `PayPalClient` and `PayPalClientError` imported above, and then `PaypalMethods` here (lower "p" Paypal, vs "PayPal")

Reply:
Not in scope for the PR. We have tooling that helps prevent any possible issues with this (e.g. static analysis).